samples/msmq/msmqtosqlrelay/MsmqTransport_1/NativeMsmqToSql/NativeMsmqToSql.csproj

```diff
@@ -8,5 +8,6 @@
     <Reference Include="System.Messaging" />
     <Reference Include="System.Transactions" />
     <PackageReference Include="Newtonsoft.Json" Version="10.*" />
+	<PackageReference Include="Microsoft.Data.SqlClient" Version="5.*" />
```

Comment:
Could we lift here the json version? 10 is rather old

Reply:
Updated to use 13